hack/update-generated-stable-metrics.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file is not intended to be run automatically. It is meant to be run
+# immediately before exporting docs. We do not want to check these documents in
+# by default.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+source "${KUBE_ROOT}/test/instrumentation/stability-utils.sh"
+
+kube::update::stablemetrics
```

Comment:
I tried changing an existing metric from ALPHA to STABLE and running this script:

```diff
diff --git a/pkg/kubelet/certificate/kubelet.go b/pkg/kubelet/certificate/kubelet.go
index af2d21717dc..0c8ef588347 100644
--- a/pkg/kubelet/certificate/kubelet.go
+++ b/pkg/kubelet/certificate/kubelet.go
@@ -82,7 +82,7 @@ func NewKubeletServerCertificateManager(kubeClient clientset.Interface, kubeCfg
 				31104000,  // 1  year
 				124416000, // 4  years
 			},
-			StabilityLevel: compbasemetrics.ALPHA,
+			StabilityLevel: compbasemetrics.STABLE,
 		},
 	)
 	legacyregistry.MustRegister(certificateRotationAge)
```

```
$ hack/update-generated-stable-metrics.sh 
go: gopkg.in/yaml.v2 upgrade => v2.4.0
./pkg/kubelet/certificate/kubelet.go:70:4: Non string attribute it not supported
exit status 1
!!! [0208 16:47:02] Call tree:
!!! [0208 16:47:02]  1: hack/update-generated-stable-metrics.sh:28 kube::update::stablemetrics(...)

$ echo $?
1

$ git status
On branch so-stable
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   pkg/kubelet/certificate/kubelet.go

no changes added to commit (use "git add" and/or "git commit -a")
```

should that have updated `test/instrumentation/testdata/stable-metrics-list.yaml`?

Reply:
Ah, yeah, Marek is tracking that one. There are some corner cases we don't support yet due (i.e. having a constant populating the subsystem field.

ref: https://github.com/kubernetes/kubernetes/issues/81863